Authors.md

```diff
@@ -8,4 +8,5 @@
 - Smit Shah (shah.smit@hotmail.com)
 - Austin Slobodnik (austinslobodnik@gmail.com)
 - Osman Findik (osmanfindik1@windowslive.com)
+- Dhruv Tailor (dhruv6198@gmail.com)
```

Comment:
Please move this below Maha, the list is chronological based on first commit for an author